Source/WebKit/UIProcess/WebAuthentication/WebAuthenticatorCoordinatorProxy.h

```diff
@@ -105,8 +105,8 @@ class WebAuthenticatorCoordinatorProxy : public IPC::MessageReceiver {
 
 #if HAVE(WEB_AUTHN_AS_MODERN)
     RetainPtr<ASAuthorizationController> constructASController(WebAuthenticationRequestData&&);
-    RetainPtr<NSArray> requestsForRegisteration(const WebCore::PublicKeyCredentialCreationOptions&, const Vector<uint8_t>& hash);
-    RetainPtr<NSArray> requestsForAssertion(const WebCore::PublicKeyCredentialRequestOptions&, const Vector<uint8_t>& hash, std::optional<WebCore::SecurityOriginData>& parentOrigin);
+    RetainPtr<NSArray> requestsForRegistration(const WebCore::PublicKeyCredentialCreationOptions&, const Vector<uint8_t>& hash, const String& origin);
```

Comment:
These are going to conflict with https://github.com/WebKit/WebKit/pull/23750

Reply:
Yeah I'll rebase once that's landed. I think that will actually remove the need for most of this PR.

Reply:
Rebased.